src/main/java/eu/dissco/orchestration/backend/security/JwtAuthConverterProperties.java

```diff
@@ -0,0 +1,15 @@
+package eu.dissco.orchestration.backend.security;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@Configuration
+@ConfigurationProperties(prefix = "jwt.auth.converter")
+public class JwtAuthConverterProperties {
+  private String resourceId;
```

Comment:
Added

Reply:
Hmm so we don't actually use these properties at the moment, I think they're meant to be used for implementing roles with keycloak. As it is, they're set to null, and our authentication still works (because we don't use role-based authentication). Will have a think on it, might want to delete these properties altogether